plugins/kubectl-casskop

```diff
@@ -20,6 +20,11 @@ import json
 from subprocess import check_output, STDOUT, CalledProcessError
 from os.path import basename
 from random import shuffle
+from itertools import groupby
+
+PAUSE_LABELS_OP = ["ToDo", "Paused"]
+NO_PODS_FOUND = "No pods found for operation"
+NO_ONGOING_OP = "--selector=operation-status notin (Ongoing, Finalizing)"
```

Comment:
not in

Reply:
It’s not python here ^ but a selector filter expression.
The syntax is `notin` and not `not in` 🤷‍♂️

Reply:
ok :)